README.md

```diff
@@ -124,10 +124,11 @@ python setup.py install
 
 =======
 ### Supported Versions
-| APEX Version  | PyTorch Version |
-| ------------- | ------------- |
-| release/1.0.0 | release/2.0 and older |
-| release/1.1.0 | release/2.1 |
+| ``APEX Version`` | ``APEX branch``  | ``Torch Version`` |
+| ------------- | ------------- | ------------- |
+| ``1.2.0`` | master | ``2.2v`` | 
+| ``1.1.0`` | release/1.1.0 | ``2.1v`` |
+| ``1.0.0`` | release/1.0.0 | ``2.0v`` and older |
```

Comment:
nit: why the "v" after 2.2, 2.1 etc? We should remove it

Reply:
I updated the file

Reply:
no particular reason, I felt it looked good :)